### ISSUE TITLE
pwa does not work offline

### ISSUE THREAD

**Issue by lallenfrancisl** [subins2000/WebDrop#22]
Since the website already works as a pwa I think it would be nice to make it work fully offline too.

**Comment by subins2000** [comment]
To discover other devices, internet is needed (using IP address to make a room name). So it's not exactly an offline app, but caching has its benefits too. Leaving this open if someone wants to do it

**Comment by lallenfrancisl** [comment]
Yeah, that's true about needing network to work. But what I was thinking was that instead of showing a offline page it would be nice to show the homepage itself and just let the user know you need network to share, you know like what they do with other similar native apps like xender while it existed.

**Comment by lallenfrancisl** [comment]
I will try to work on this coming weekend or week after freeing up some time
